test/test_sbol2_sbol3_direct.py

```diff
@@ -12,6 +12,15 @@
 TEST_FILES = Path(__file__).parent / 'test_files'
 
 
+class SBOLValidationError(Exception): pass
```

Comment:
I'd suggest adding a comment about how this is distinct from `sbol3.ValidationError` and `sbol2.SBOLError`.